examples/benchmark/benchmark.py

```diff
@@ -41,7 +43,7 @@ def main():
             chip.set('flowarg', 'syn_np', n)
             chip.set('flowarg', 'place_np', n)
             chip.set('flowarg', 'cts_np', n)
-            chip.load_flow('asicflow')
+            chip.use(asicflow)
```

Comment:
Isn't the skywater130_demo loading the asicflow already?

Reply:
Yes, but I think that this example might re-load it to have multiple parallel tasks for syn, place, and cts.

Reply:
You are right!